refactor(curriculum): improve binary search tree (#53622)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/07-scientific-computing-with-python/learn-tree-traversal-by-building-a-binary-search-tree/65ca0d5adf39c410cd1177cc.md

---
id: 65ca0d5adf39c410cd1177cc
title: Step 50
challengeType: 20
dashedName: step-50
---

# --description--

> Now, you would work on traversing the tree based on the in-order traversal method. In-order traversal is a depth-first binary tree traversal algorithm that visits the left subtree, the current node, and then the right subtree.

Reviewer: ```suggestion
Now, you'll work on traversing the tree based on the in-order traversal method. In-order traversal is a depth-first binary tree traversal algorithm that visits the left subtree, the current node, and then the right subtree.
```
